sections/en/being_afraid_of_other_peoples_code.md

```diff
@@ -1,37 +1,37 @@
 
-This is however a strange mentality, mainly encountered amongst web developers in the PHP community, it's one that exudes a lack of professionalism and experience.
+This is however a strange mentality, mainly encountered amongst web developers in the PHP community. It's one that exudes a lack of professionalism and experience.
 
-Writing software and dealing with other peoples code is normal, it's part of the daily work of a professional programmer, it's not something to be afraid of.
+Writing software and dealing with other people's code is normal. It's part of the daily work of a professional programmer. It's not something to be afraid of.
 
-A professional programmer doesn't look at other peoples code and start whining about how he or she is at the complete mercy of the former programmer, who perhaps is no longer associated with the company or project, and if only the former programmer had used framework A or framework B the day would have been saved.
+A professional programmer doesn't look at other people's code and start whining about how he or she is at the complete mercy of the former programmer, who is perhaps no longer associated with the company or project, and if only the former programmer had used framework A or framework B the day would have been saved.
 
 This is not the mentality of a professional programmer. Nobody does this.
 
 Perhaps the low barrier to entry in PHP web development plays a part in this kind of mentality. Regardless, it is a sign of a person being in the wrong line of work.
 
-A great part of programming deals with people having to work with other peoples code. It's part of the work trying to improve existing codebase and sometimes that involves a complete re-write.
+A great part of programming deals with people having to work with other people's code. It's part of the work trying to improve existing codebase and sometimes that involves a complete re-write.
 
 Take note from the great masters of programming, read the book [Coders at work - Reflections on the Craft of Programming](http://codersatwork.com/).
 
 Some of the largest and most successful codebases in world are codebases that has been developed by hundreds of people who have never even met each other, codebases developed without the use of any kind of framework, codebases done entirely in a procedural programming language without the use of anything but the procedural paradigm, and they wouldn't dream of doing it differently.
 
-The [Linux Kernel](https://www.kernel.org/) consist of more than 20 million lines of code all written entirely using procedural programming by more than 14.000 participants without the usage of any kind of framework.
+The [Linux Kernel](https://www.kernel.org/) consists of more than 20 million lines of code all written entirely using procedural programming by more than 14,000 participants without the use of any kind of framework.
```

Comment:
While a comma (`,`) is the most common in the English speaking world, a thin space (` `) is the _de facto_ international standard for digit grouping. The full stop (`.`) is common throughout non-English-speaking Europe.

Just throwing it out that his use of the full stop isn't wrong, so much as it's simply uncommon.